client/extensions/woocommerce/app/settings/payments/payment-method-edit.js

```diff
@@ -0,0 +1,106 @@
+/**
```

Comment:
This component seems not payment method specific at all. This might be a good candidate for woocommerce/components/ for use with settings-like things in general.